kraken/time_actions/common_time_functions.py

```diff
@@ -134,3 +136,17 @@ def check_date_time(object_type, names):
             if counter < max_retries:
                 logging.info("Date in pod " + str(pod_name[0]) + " reset properly")
     return not_reset
+
+
+def run_scenarios(scenarios_list, config, wait_duration):
```

Comment:
sorry one more inconsistent name of the main function.

Reply:
Fixed it, thanks.